src/test-routes/reserved-words.ts

```diff
@@ -23,4 +23,15 @@ app.category("vanilla", () => {
   app.put("/reservedWords/operation/files", "reservedWordsBodyNamedFiles", (req) => {
     return { status: 200 };
   });
+  app.put("/reservedWords/foo", "reservedWordsUrlHeaderQuery", (req) => {
+    if (req.query["queryParameters"][0] !== "one" && req.query["queryParameters"][1] !== "two") {
```

Comment:
@timotheeguerin based off of logging, I can see that `req.query["queryParameters"]` should be a list of ["one", "two"], but checking it as a list isn't quite working, it fails the check. It also says that `req.query]["queryParameters"]` is an object, i'm not sure if this is maybe because everything is kind of an object in ts

Reply:
what does the logging logs exactly? might be an object {0: "one", 1: "two"} so might needs quotes to check it.

Reply:
No, basically if I do a validation error and validate that `req.query["queryParameters"] == ["one", "two"]`, I get

```
warn: Request validation failed: The query parameters your passed in were incorrect:
 Expected:
 one,two
 Actual: 
one,two
```

The only thing is the spacing is different

Reply:
and the value isn't just a string?

Reply:
the spacing might be a problem with the formatting of  the error.
If you do console.log(`Value is "${req.query["queryParameters"]}"`) what do you get

Reply:
I get `Value is "one,two"`. However, if I set the value to "one,two" in my check, it says the same thing as if I do it as a list

Reply:
So this doesn't work?
```ts
if (req.query["queryParameters"] !== "one,two") {
 throw ...
}

```